drivers/hwinfo/hwinfo_smartbond.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2023 Jerzy Kasenberg.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/hwinfo.h>
+#include <DA1469xAB.h>
```

Comment:
This looks very particular too me. Usually we include soc.h

Reply:
changed to soc.h